build_tools/circle/build_doc.sh

```diff
@@ -116,7 +116,7 @@ conda update --yes --quiet conda
 # provided versions
 conda create -n $CONDA_ENV_NAME --yes --quiet python="${PYTHON_VERSION:-*}" \
   numpy="${NUMPY_VERSION:-*}" scipy="${SCIPY_VERSION:-*}" \
-  pytest coverage matplotlib="${MATPLOTLIB_VERSION:-*}" sphinx \
+  pytest coverage matplotlib="${MATPLOTLIB_VERSION:-*}" sphinx=* \
```

Comment:
```suggestion
  pytest coverage matplotlib="${MATPLOTLIB_VERSION:-*}" sphinx \
```
We do not need this here. I guess it is already fixed.